index.html

```diff
@@ -313,10 +328,10 @@ <h2>
                 Languages
             </h2>
             <ul class="eylemm">
-                <li><button class="btn2">JavaScript</button></li>
-                <li><button class="btn2">Ruby</button></li>
-                <li><button class="btn2">Html</button></li>
-                <li><button class="btn2">CSS</button></li>
+                <li><button type="button" class="btn2">JavaScript</button></li>
+                <li><button type="button" class="btn2">Ruby</button></li>
+                <li><button type="button" class="btn2">Html</button></li>
+                <li><button type="button" class="btn2">CSS</button></li>
             </ul>
         </div>
```

Comment:
- [ ] For the recent `About Myself` section, kindly make this section look identical to that of the Figma template you chose. To highlight the changes to be made in this section, you need to:

- Make the `Languages`, `Framework` and `Skills` subsections align on a row. 
- Use the background images used for this section and align the accordingly
- Alignment and structure of the `p` element with the text `"I can help you build a product ..."`
- The alignment of each card and its badges. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/208153934-9992c382-b833-4c3e-bca9-831148117902.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/208153801-56e21544-db4e-4835-aed5-c0f0f7ccb8d3.png)


#### Hint: Use the `grid` to style the section and then `flexbox` to style individual cards 👍